.travis.yml

```diff
@@ -8,9 +8,9 @@ matrix:
   include:
     - elixir: 1.4.5
       otp_release: 19.3
```

Comment:
We should probably also still test 1.5.x?

Reply:
In the past, we've just tested the oldest and newer supported version, but I don't feel strongly. I'll re-add a matrix entry for 1.5.